addon/MetadataStore.js

```diff
@@ -489,7 +489,7 @@ MetadataStore.prototype = {
             });
             break;
           default:
-            throw new Error("Fetched unknown image types: {image.type}");
+            throw new Error(`Fetched unknown image types: ${image.type}`);
```

Comment:
No. This is more like an "assert" here, i.e. this should never happen. If it does happen, it'll let the caller handle this exception.